7_react/react1/week2/todo-app/src/todoItem/TodoItem.jsx

```diff
@@ -0,0 +1,29 @@
+import "./todoitem.css";
+import { useState } from "react";
+
+export const TodoItem = (props) => {
+  const [itemStatus, setItemStatus] = useState(props.status);
+
+  function toggleStatus(e) {
+    setItemStatus(!itemStatus);
+  }
+
+  return (
+    <div className="list_elem">
+      <input
+        type="checkbox"
+        id="item"
+        name="scales"
+        checked={itemStatus}
+        onChange={toggleStatus}
+      />
+      <label htmlFor="item">{props.name}</label>
```

Comment:
You are missing the task to add a line-through decoration to checked item from the homework. You can conditionally add a classname like this 
`<label htmlFor="item" className={props.status ? 'done' : ''}>`
This way the label element will only have 'done' class when the props.status is true. Now only need to create CSS for it!